content/docs/SUSHI/project/_index.md

```diff
@@ -23,7 +23,7 @@ Each FSH file can contain multiple FSH definitions of varying types. FSH file na
 
 ### Ignoring Warnings
 
-SUSHI may log warnings based on the content of the project, and these warnings can be ignored with an optional **sushi-ignoreWarnings.txt** file. Note that the contents of this file can only be used to ignore warnings. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., **simple-project/sushi-ignoreWarnings.txt** in the example above), or within the **input** directory (e.g., **simple-project/input/sushi-ignoreWarnings.txt**). Each line in the **sushi-ignoreWarnings.txt** file specifies warnings to ignore.  Warnings will be ignored if they completely match the contents of any line in the **sushi-ignoreWarnings.txt** file. The warning to ignore must be specified on only one line, and the match is case-sensitive. Additionally, regular expressions can be specified in the **sushi-ignoreWarnings.txt** file. Each regular expression must be on only one line, and a regular expression is indicated by starting and ending the line with `/`. For example, if the **sushi-ignoreWarnings.txt** file was as follows:
+SUSHI may log warnings based on the content of the project, which can be ignored with an optional sushi-ignoreWarnings.txt file. Note that the contents of this file can only be used to ignore warnings. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., **simple-project/sushi-ignoreWarnings.txt** in the example above), or within the **input** directory (e.g., **simple-project/input/sushi-ignoreWarnings.txt**). Each line in the **sushi-ignoreWarnings.txt** file specifies warnings to ignore.  Warnings will be ignored if they completely match the contents of any line in the **sushi-ignoreWarnings.txt** file. The warning to ignore must be specified on only one line, and the match is case-sensitive. Additionally, regular expressions can be specified in the **sushi-ignoreWarnings.txt** file. Each regular expression must be on only one line, and a regular expression is indicated by starting and ending the line with `/`. For example, if the **sushi-ignoreWarnings.txt** file was as follows:
```

Comment:
Oh no this is completely my fault. But I think we should still bold the `sushi-ignoreWarnings.txt` reference in the first sentence.

> which can be ignored with an optional **sushi-ignoreWarnings.txt** file

Reply:
Oh la la ah oui oui. Fixed in 04872e1.